components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -59,6 +60,10 @@ abstract class EngineSession(
         fun onCloseWindowRequest(windowRequest: WindowRequest) = Unit
         fun onMediaAdded(media: Media) = Unit
         fun onMediaRemoved(media: Media) = Unit
+        /**
```

Comment:
nit: Not sure we need kdocs here, but if then we should describe it differently.

This isn't to "add a browser action", but to change/override it for this session e.g. changing the badgeTest.

Reply:
Yes, we need the comments, since our last update to detekt.yml file  https://github.com/mozilla-mobile/android-components/commit/f82a69d9f1027cb9a75fc4d29dcf73bd2132a8e0 we get an [UndocumentedPublicFunction](https://arturbosch.github.io/detekt/comments.html#undocumentedpublicfunction) if we don't do that.

Reply:
👍 we're getting stricter and stricter :)